OrbitClientGgp/ClientGgp.cpp

```diff
@@ -65,8 +65,6 @@ bool ClientGgp::RequestStartCapture(ThreadPool* thread_pool) {
   // TODO(kuebler): right now selected_functions is only an empty placeholder,
   //  it needs to be filled separately in each client and then passed.
   absl::flat_hash_map<uint64_t, orbit_client_protos::FunctionInfo> selected_functions;
-  Capture::capture_data_ =
```

Comment:
Why is this removed? I still need Capture and capture_data for processing the data of the capture (which is not done yet but it is the next step)